app/controllers/home_controller.rb

```diff
@@ -7,7 +7,7 @@ def form
     user = User.new(email: params["email"])
     user.save
     session[:user]=user.id
```

Comment:
So, for your information, it is definitely good practice to keep a reference to the user in the session, but we can abstract this sort of common pattern into helpers, here's how MV Home is doing it: https://github.com/mindvalley/MVA_Home/blob/fdffdd751b43519d2cdafd8ecf7d7bf61903dab4/app/helpers/auth0_helper.rb

We can do the same once we implement authentication.